### ISSUE TITLE
The script exited abnormally when running scripts/train_sac_inner.py

### ISSUE THREAD

**Issue by beiliwzl** [CasperTeirlinck/RLFC-SACIDHP#2]
Dear CasperTeirlinck

only 3993 steps run, can you check for me, thank you very much.
I do not use the gpu, and the tensorflow is 2.7.1 

(rlfc-sacidhp1) D:\Backup\Documents\Machine Learning\PycharmProjects\gitzoo\RLFC-SACIDHP-public>python scripts/train_sac_inner.py
2025-04-05 14:10:55.919946: W tensorflow/stream_executor/platform/default/dso_loader.cc:64] Could not load dynamic library 'cudart64_110.dll'; dlerror: cudart64_110.dll not found
2025-04-05 14:10:55.920274: I tensorflow/stream_executor/cuda/cudart_stub.cc:29] Ignore above cudart dlerror if you do not have a GPU set up on your machine.
Using random seed 2786956550
2025-04-05 14:11:02.595110: E tensorflow/stream_executor/cuda/cuda_driver.cc:271] failed call to cuInit: CUDA_ERROR_NO_DEVICE: no CUDA-capable device is detected
2025-04-05 14:11:02.603128: I tensorflow/stream_executor/cuda/cuda_diagnostics.cc:169] retrieving CUDA diagnostic information for host: Desktop-Beiliwzl
2025-04-05 14:11:02.603426: I tensorflow/stream_executor/cuda/cuda_diagnostics.cc:176] hostname: Desktop-Beiliwzl
2025-04-05 14:11:02.603897: I tensorflow/core/platform/cpu_feature_guard.cc:151] This TensorFlow binary is optimized with oneAPI Deep Neural Network Library (oneDNN) to use the following CPU instructions in performance-critical operations:  AVX AVX2
To enable them in other operations, rebuild TensorFlow with the appropriate compiler flags.
Training SAC
Saving to trained/SAC_citation_tracking_attitude_1743833463
  0%|                                                                                                | 0/1000000 [00:00<?, ?it/s, ep=1, crashes=0, crashes_eval=0, return_avg=0, return_best=-inf, nmae_best=inf%]2025-04-05 14:11:03.551529: I tensorflow/compiler/xla/service/service.cc:171] XLA service 0x2175c8346a0 initialized for platform Host (this does not guarantee that XLA will be used). Devices:
2025-04-05 14:11:03.552132: I tensorflow/compiler/xla/service/service.cc:179]   StreamExecutor device (0): Host, Default Version
2025-04-05 14:11:03.583039: I tensorflow/compiler/mlir/tensorflow/utils/dump_mlir_util.cc:237] disabling MLIR crash reproducer, set env var `MLIR_CRASH_REPRODUCER_DIRECTORY` to enable.
2025-04-05 14:11:03.876587: I tensorflow/compiler/jit/xla_compilation_cache.cc:351] Compiled cluster using XLA!  This line is logged at most once for the lifetime of the process.
  0%|                                                                                     | 234/1000000 [00:01<51:22, 324.32it/s, ep=1, crashes=0, crashes_eval=0, return_avg=0, return_best=-inf, nmae_best=inf%]2025-04-05 14:11:09.390455: W tensorflow/compiler/tf2xla/kernels/assert_op.cc:38] Ignoring Assert operator assert_positive/assert_less/Assert/Assert
2025-04-05 14:11:09.391635: W tensorflow/compiler/tf2xla/kernels/assert_op.cc:38] Ignoring Assert operator assert_positive_1/assert_less/Assert/Assert
  0%|▎                                                                    | 3993/1000000 [00:55<2:17:46, 120.48it/s, ep=2, crashes=0, crashes_eval=0, return_avg=-1.01e+3, return_best=-1.01e+3, nmae_best=33.00%]

**Comment by beiliwzl** [comment]
I try to upgrade the tensorflow to 2.12.0, then it works.
